charts/Chart.yaml

```diff
@@ -49,8 +49,6 @@ annotations:
   helm.sh/images: |
     - name: etcd
       image: docker.io/bitnami/etcd:3.6.2-debian-12-r0
-    - name: kubectl
-      image: docker.io/bitnami/kubectl:1.25.15
```

Comment:
Need to modify the helm images scripts to make sure we generate the script, we don't want to miss on this image...